### PR TITLE
FE sample update request

### PR DESCRIPTION
**Pull request by phoenixAja**
### Summary:
- **What:** adding in sample put request to backend
- **Ticket:** [sc183511](https://app.shortcut.com/genepi/story/183511)
- **Env:**  no rdev

### Checklist:
- [x] I merged latest `trunk`
- [x] I manually verified the change
- [x] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/frontend/src/common/api/index.tsx [hunk 1]
```diff
@@ -162,6 +163,19 @@ export async function getBackendApiJson<T>(
   return await makeBackendApiJsonCall(route, requestOptions);
 }
 
+export async function putBackendApiJson<T>(
```
Comment (vincent-czi) [inline src/frontend/src/common/api/index.tsx:166]: Thank you!

File: src/frontend/src/common/api/index.tsx [hunk 2]
```diff
@@ -29,6 +29,7 @@ export const DEFAULT_FETCH_OPTIONS: RequestInit = {
 export const DEFAULT_PUT_OPTIONS: RequestInit = {
   credentials: "include",
   method: "PUT",
+  ...DEFAULT_HEADERS_MUTATION_OPTIONS,
```
Comment (vincent-czi) [inline src/frontend/src/common/api/index.tsx:32]: Thank you! Minor additional request:
Could you go to `src/frontend/src/common/queries/auth.ts` and remove the matched `DEFAULT_HEADERS_MUTATION_OPTIONS` spread in that file? It was there to accomplish the same thing in a specific way that is now being done in a general way with your change here of including  `DEFAULT_HEADERS_MUTATION_OPTIONS` inside of `DEFAULT_PUT_OPTIONS`. Please and thank you!

**Review by vincent-czi — APPROVED** [review]
Looks good! Thanks for the PUT improvements!

File: src/frontend/src/common/api/index.tsx (hunk 2 above)
Comment (phoenixAja) [reply src/frontend/src/common/api/index.tsx:32]: oh yes good catch will do!

**Pull request merged**